pkg/controller/common/metrics.go

```diff
@@ -0,0 +1,67 @@
+package common
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	// DefaultBindAddress is the port for the metrics listener
+	DefaultBindAddress = ":8797"
+
+	// MCCImportantConfigPaused logs when important config is being held up by pause
+	MCCImportantConfigPaused = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "mcc_important_config_paused",
```

Comment:
I think about the machine-config controller often enough to recognize the `mcc` acronym.  But many folks browsing available metrics on their cluster will not.  Can we write this out `machine_config_controller_content_paused` or some such?